.gitignore

```diff
@@ -49,3 +49,7 @@ src/components/icon/generated-icons.ts
 /playwright-report/
 /blob-report/
 /playwright/.cache/
+
+
+/public/robots.txt
+/public/sitemap.xml
```

Comment:
Why are these ignored?

Reply:
As i understood it, these files are automatically be created postbuild, and are therefore not necessary to push.